medusa/server/api/v2/scene_exception.py

```diff
@@ -16,20 +16,67 @@ def get_last_updates():
     return last_updates
 
 
-class SceneExceptionOperationHandler(BaseRequestHandler):
-    """Scene Exception operation request handler."""
+class SceneExceptionTypeHandler(BaseRequestHandler):
+    """Scene Exception type request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionTypeHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET')
+
+    def get(self, exception_type):
+        if exception_type is not None and exception_type not in ['medusa', 'xem', 'anidb']:
```

Comment:
While we're talking about naming exceptions, instead of `SceneExceptions` what do you think about calling them `NamingExceptions`?  They don't only apply to scene, and they aren't exceptions to Scene, so it might be a little more appropriate.